src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -1149,15 +1149,21 @@ class TcgStatics {
 
   /* General checks for attacks and abilities */
 
-  //  * assertBench
-  //    - Can be used with no arguments, or with a condition and additional text included.
-  //    - If a closure is given, it'll check for any "it" element in bench returning true.
-  //    - Optional params:
-  //      + isType: if set, restricts to benched Pokémon of a single specific type.
-  //      + hasPokemonEX/hasPokemonGX/hasPokemonV: Can be expanded if needed. All of these unset will have the method search for any Pokémon no matter what, but if even a single one is set true it'll only filter those that are set true as well.
-  //      + opp: If true, checks for the opponent's bench instead of "my" bench.
-  //      + repText: If true, cText will override the entire fail message instead of being added at the end.
-  static void assertBench(params=[:], Closure c = null, String cText = null) {
+  /**
+   *
+   * Does a customized assert with a automated fail message.
+   *
+   * @param params Optional settings that can be added:
+   *   + isType: if set, restricts to benched Pokémon of a single specific type.
+   *   + hasPokemonEX/hasPokemonGX/hasPokemonV: Can be expanded if needed. All of these unset will have the method search for any Pokémon no matter what, but if even a single one is set true it'll only filter those that are set true as well.
+   *   + opp: If true, checks for the opponent's bench instead of "my" bench.
+   *
+   * @param c Additional condition the filtered benched Pokémon must follow. Defaults to true (so any benched Pokémon).
+   *
+   * @param message Additional text to be put at the end of the assert fail warning. If repText is set to true in params, message will override the entirety of said warning.
+   *
+   */
+  static void assertBench(params=[:], Closure c = null, String message = "") {
```

Comment:
Hmm, I put it like that based on other commands using info as an explicit parameter. Which looking at it might be a better name for it since it won't be a full message 99% of the time.

Reply:
I just pushed a correction on that, also ignoring all the type and pokemon logic if the warning is fully overrode by `repText: true`. Will update the doc in a bit.